src/libwalletqt/WalletManager.cpp

```diff
@@ -86,6 +86,21 @@ Wallet *WalletManager::recoveryWallet(const QString &path, const QString &memo,
     return m_currentWallet;
 }
 
+Wallet *WalletManager::createWalletFromKeys(const QString &path, const QString &language, bool testnet,
+                                            const QString &address, const QString &viewkey, const QString &spendkey,
+                                            quint64 restoreHeight)
+{
+    QMutexLocker locker(&m_mutex);
+    if (m_currentWallet) {
+        qDebug() << "Closing open m_currentWallet" << m_currentWallet;
+        delete m_currentWallet;
```

Comment:
ah. It's actually a guarded pointer, so I don't think it's needed?
http://doc.qt.io/qt-5/qpointer.html